src/SFA.DAS.EmployerIncentives.Api/AppStart/AddServiceRegistrationExtension.cs

```diff
@@ -23,6 +26,12 @@ public static void AddServiceRegistration(this IServiceCollection services)
             services.AddTransient<IEmployerIncentivesService, EmployerIncentivesService>();
             services.AddTransient<ICommitmentsService, CommitmentsService>();
             services.AddTransient<IAccountsService, AccountsService>();
+
+            int maxParallelism = 100;
+            var throttler = Policy.BulkheadAsync<HttpResponseMessage>(maxParallelism, int.MaxValue);
+
+            services.AddHttpClient("Throttled")
+                .AddPolicyHandler(throttler);
```

Comment:
perhaps link this to the Client that you want this applying to:

```
int maxParallelism = 100;
            var throttler = Policy.BulkheadAsync<HttpResponseMessage>(maxParallelism, int.MaxValue);

            services.AddHttpClient(nameof(EmployerIncentivesConfiguration))
                .AddPolicyHandler(throttler);
```

And then change the APIClient to the below 👇🏻